src/analytics/Events.tsx

```diff
@@ -438,6 +438,9 @@ export enum FiatExchangeEvents {
   cico_submit_transfer = 'cico_submit_transfer',
   cico_cancel_transfer = 'cico_cancel_transfer',
 
+  // Fiat Connect link account screen
```

Comment:
Should we add an event for when a user presses back on this screen so we can track dropout at this part of the funnel?